rfcs/tag-component.md

```diff
@@ -0,0 +1,61 @@
+# Tag Component
+
+### Exports
+
+- `Tag`
+- `StyledRoot`
+- `StyledCross`
+
+### `Tag` API
+
+- `color: STYLE.blue(theme.colors.primary470) | STYLE.orange(theme.colors.warning400) | STYLE.green(theme.colors.positive400) | STYLE.red(theme.colors.negative400)`:
```

Comment:
Is it worth introducing semantics here rather than color names? i.e. `STYLE.info, STYLE.warning, STYLE.positive, STYLE.negative`

Reply:
semantics allow user to choose predefined default styles rather than going to theme and trying to find corresponding color, which is not so clear